website/blog/2023-05-14-databend-weekly-93.md

```diff
@@ -0,0 +1,126 @@
+---
+title: "This Week in Databend #93"
+date: 2023-05-14
+slug: 2023-05-14-databend-weekly
+tags: [databend, weekly]
+description: "Stay up to date with the latest weekly developments on Databend!"
+contributors:
+  - name: andylokandy
+  - name: b41sh
+  - name: BohuTANG
+  - name: Chasen-Zhang
+  - name: dantengsky
+  - name: DongHaowen
+  - name: Dousir9
+  - name: drmingdrmer
+  - name: everpcpc
+  - name: flaneur2020
+  - name: hantmac
+  - name: lichuang
+  - name: Mehrbod2002
+  - name: PsiACE
+  - name: SkyFan2002
+  - name: soyeric128
+  - name: sundy-li
+  - name: TCeason
+  - name: Xuanwo
+  - name: xudong963
+  - name: youngsofun
+  - name: zhang2014
+  - name: ZhiHanZ
+  - name: zhyass
+authors:
+  - name: PsiACE
+    url: https://github.com/psiace
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+> The `meta-upgrade-09` will not be available in the upcoming Databend release. If you are currently using version 0.9 or an earlier version, please utilize our upgrade tool which can be found at: [v1.1.33-nightly](https://github.com/datafuselabs/databend/releases/tag/v1.1.33-nightly).
```

Comment:
The upgrade tool meta-upgrade-09 will no longer be available in the release package. If you're using Databend 0.9 or an earlier version, you can seek help from the community.